### PR TITLE
sshguard: patch for nginx support & fix init script

### PR DESCRIPTION
**Pull request by alip**

### CODE REVIEW

File: app-admin/sshguard/files/sshguard.initd [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/sbin/openrc-run
+# Copyright 1999-2011 Gentoo Foundation
```
Comment (jolenzy) [inline app-admin/sshguard/files/sshguard.initd]: 2017? :)

Comment (alip) [inline app-admin/sshguard/files/sshguard.initd]: :) updating.

File: app-admin/sshguard/sshguard-2.0.0-r100.ebuild [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
+
+DESCRIPTION="protects hosts from brute force attacks against ssh"
```
Comment (jolenzy) [inline app-admin/sshguard/sshguard-2.0.0-r100.ebuild]: Capital P? :)

**Comment by jolenzy** [comment]
My very minor suggestions @alip  :)
Looks good to me :+1:

**Comment by alip** [comment]
Thanks, both fixed. I'll go ahead and merge.

**Pull request merged**